commons/src/main/java/com/powsybl/commons/extensions/ExtensionSeriesSerializer.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.extensions;
+
+import java.util.Map;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public interface ExtensionSeriesSerializer<T extends Extendable, E extends Extension<T>> extends ExtensionProvider<T, E> {
+
+    int STRING_SERIES_TYPE = 0;
```

Comment:
I think here we should make this an enum:
we'll convert to an `int` in pypowsybl